tools/utils/download_blender.sh

```diff
@@ -64,20 +73,64 @@ function get_extractor() {
     echo "${extractor}"
 }
 
+function verify_download_integrity() {
+    local version=${1}
+    local target_filepath=${2}
+
+    if [ ! -f "${target_filepath}" ]; then
+        return 1
+    fi
+
+    echo "Found Blender ${ver} download. Verifying the integrity."
+
+    local target_filename checksum_url checksum_url download_dir
```

Comment:
There are two `checksum_url`, so we can remove one.
Or, maybe this should be replaced by `checksum_filename`.

Reply:
Good spot!